Assets/Editor/BuildTiltBrush.cs

```diff
@@ -961,6 +961,10 @@ public TempSetAppNames(bool isAndroid, string Description)
             m_company = PlayerSettings.companyName;
             string new_name = App.kAppDisplayName;
             string new_identifier = GuiBuildAndroidApplicationIdentifier;
+#if OCULUS_SUPPORTED
+            //Can't change Quest identifier
+            new_identifier = "com.Icosa.OpenBrush";
+#endif
```

Comment:
Does this make it into the manifest? If so, can we remove that package=XXX completely? If not, is this a problem?

Reply:
I'll check the CI when it finishes, but yes this should override the manifest template, per my mistake in #280 that was fixed with #281

Reply:
Yes looks to have worked.

Reply:
awesome! Should we replace the one in the manifest with something like `com.to.be.replaced.at.build.time`?

Reply:
We could probably remove the line entirely, I could give that a shot!